docs/manual/tags.md

```diff
@@ -298,22 +298,22 @@ Transfiletriggerversion     | 5081 | string array
 
 ## Signatures and digests
 
-[Signatures](signatures.md) allow to verify the origin of a package.
+[Signatures](signatures_digests.md) allow verifying the origin of a package.
 
 Tag Name          | Value| Type         | Description
 ------------------|------|--------------|------------
-Dsaheader         | 267  | bin          | OpenPGP DSA signature of the header (if thus signed)
-Longsigsize       | 270  | int64        | Header+payload size if > 4GB.
+Dsaheader         | 267  | bin          | OpenPGP EdDSA signature of the header (if thus signed)
```

Comment:
This needs to be "DSA or EdDSA" really, EdDSA is a *very* new addition in the v4 timescale.